.goreleaser.yml

```diff
@@ -1,9 +1,9 @@
 # Copyright 2023 Nutanix. All rights reserved.
 # SPDX-License-Identifier: Apache-2.0
 
-version: 2
+# yaml-language-server: $schema=https://goreleaser.com/static/schema.json
 
-project_name: cluster-api-runtime-extensions-nutanix
```

Comment:
did this removed accidentally? goreleaser can inferred it. however this change seems unrelated.